examples/two_dimensional_turbulence.jl

```diff
@@ -1,34 +1,27 @@
 # # Two dimensional turbulence example
 #
-# In this example, we initialize a random velocity field and observe its viscous,
-# turbulent decay in a two-dimensional domain. This example demonstrates:
+# In this example, we initialize a random velocity field and observe its turbulent decay 
+# in a two-dimensional domain. This example demonstrates:
 #
 #   * How to run a model with no buoyancy equation or tracers;
 #   * How to create user-defined fields
 #   * How to use differentiation functions
+#   * How to save a computed field
 
 # ## Model setup
 
-# For this example, we need `Plots` for plotting and `Statistics` for setting up
-# a random initial condition with zero mean velocity.
+using Oceananigans, Oceananigans.Advection
 
-using Plots, Statistics
-
-# In addition to importing plotting and statistics packages, we import
-# `Oceananigans`, its `AbstractOperations` and `Grids` submodules,
-# and the `Face` and `Cell` types to will aid in the calculation
-# and visualization of voriticty.
-
-using Oceananigans, Oceananigans.AbstractOperations, Oceananigans.Grids
-
-# `Face` and `Cell` represent "locations" on the staggered grid. We instantiate the
-# model with a simple isotropic diffusivity.
+# We instantiate the model with a simple isotropic diffusivity. We also use a 4-th order 
+# advection scheme and Runge-Kutta 3rd order time-stepping scheme.
 
 model = IncompressibleModel(
-        grid = RegularCartesianGrid(size=(128, 128, 1), extent=(2π, 2π, 2π)),
+        grid = RegularCartesianGrid(size=(128, 128, 1), extent=(2π, 2π, eps(Float64))),
```

Comment:
```suggestion
        grid = RegularCartesianGrid(size=(128, 128, 1), halo=(2, 2, 2), extent=(2π, 2π, eps(Float64))),
```

Ah `CenteredFourthOrder` needs halos of size 2. This should fix the failing test.

This should be made more automatic and user-friendly. See #941.

Reply:
Hmmm, I wonder if it's a good use to use `eps(Float64)` in a beginner example. Maybe we should wait until #1003 is merged to get rid of the `eps(Float64)` and make the example more user-friendly?

Reply:
oh that was a mistake! I didn't mean to push `eps(Float64)`!!